plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
```

Comment:
If the service name is different, maybe it should be made the same for both. I don't know if it was intentional. @grdryn  Can you better answer @jessesarn question? Perhaps the goal is to realign them as part of (private issue) https://issues.jboss.org/browse/RHMAP-7960